mods/tuxemon/db/npc/taba_acolyte_1.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
if you decide to add the acolyte monsters in map, then you can remove the monsters from here.